lib/widget/game.dart

```diff
@@ -435,24 +525,43 @@ class GameActiveViewModel {
 class SecretBoardModel {
   final Player player;
   final Set<String> visibleToAccountant;
+  final String kingpinGuess;
```

Comment:
Comment 3: Adding kingpinGuess to the model.

I've also fixed equals, hashcode and toString, where I had forgotten to add the new values.